rust_src/src/lisp.rs

```diff
@@ -506,3 +506,15 @@ pub fn XMARKER(a: LispObject) -> *const LispMarker {
 fn test_markerp() {
     assert!(!MARKERP(Qnil))
 }
+
+// Implementation of the XFASTINT depends on the USE_LSB_TAG
+// in Emacs C. But we selected this implementation as in our
+// build that value is 1.
+// A must be nonnegative.
+#[allow(dead_code)]
+#[allow(non_snake_case)]
+fn XFASTINT(a: LispObject) -> EmacsInt {
```

Comment:
Yes, sure, updated. Moved it into `depreacted` mod in list.rs, did I understand you correctly?